### DISCUSSION TITLE
wechat-uos 的问题

### DISCUSSION THREAD

**Discussion by abfactor** [fcitx/fcitx5#729] [Q&A] [answered]
在使用wechat-uos的时候出现的问题：
聊天框正常使用拼音如：
输入 nihao<回车>
输入框正确的输入了nihao

如果是输入 n<回车>
输入会直接把回车也提交给 wechat-uos 消息就直接发送出去了。

在需要输入单个字母的时候非常麻烦：比如"50m"按下m<回车后>消息直接发送了。

不清楚是wechat-uos的bug还是fcitx5的问题。

[wechat-uos aur](https://aur.archlinux.org/packages/wechat-uos)

**Comment by wengxt** [comment, marked as answer]
是否是单个字母从fcitx的角度没有任何区别，因此不是fcitx的bug。

事实上我曾经给chromium报过一个很类似的bug

https://bugs.chromium.org/p/chromium/issues/detail?id=1278989

我怀疑bug的原理是chrome对输入法提交的单个字母有特殊处理有可能内部被转换成按键
